SpotifySearch/SpotifySearch/CellController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CellController.swift
+//  SpotifySearch
+//
+//  Created by Marty Avedon on 10/28/16.
+//  Copyright © 2016 Marty Hernandez Avedon. All rights reserved.
+//
+
+import UIKit
+
+class CellController: UITableViewController {
```

Comment:
`CellController` is not a great for a TableView Controller. Something like `SpotifyTableViewController` would be simple and acceptable.